public/css/game.scss

```diff
@@ -0,0 +1,186 @@
+// scss-lint:disable IdSelector, PropertySortOrder, ColorVariable
+.container {
+  width: 100%;
+  margin: 0 !important;
+}
+
+.game-cards {
+  height: 180px;
+  border: 1px solid black;
+  width: 130px;
+  margin: 50px;
+  background-color: black;
+  border-radius: 5px;
+  color: white;
+  p {
+    font-size: smaller;
+  }
+}
+
+.players-cards {
+  height: 500px;
+}
+
+.deck-of-black-cards {
+  box-shadow: -2px -2px 0px 0px  #000000;
+  border-top: 1px solid #ebebe0;
+  border-left: 1px solid #ebebe0;
+  line-height: 180px;
+  text-align: center;
+  font-size: 25px;
+}
+
+.black-card {
+  padding: 10px;
+  height: 200px;
+  width: 150px;
+  font-size: medium;
+  u {
+    position: relative;
+    content: '';
+    display: inline-block;
+    width: 25%;
+    text-decoration: none;
+    height: 1em;
+    border-bottom: 0.125em solid white;
+    margin-bottom: -0.125em;
+  }
+}
+
+.start-game, .awaiting-players {
+  margin-top: 100px;
+}
+
+#start-game p {
+  margin-left: 10px;
+}
+
+.question-holder {
+  padding-left: 100px;
+}
+
+.black-cards-bar {
+  height: 300px;
+  border-bottom: 3px solid #eee;
+}
+
+#white-cards-bar {
+  border-left: 3px solid #eee;
+}
+
+.players-bar {}
```

Comment:
why do you have a class selector with an empty style? do remove it if you are not adding styles for that class. ;)